.gitignore

```diff
@@ -33,3 +33,4 @@ out/
 ehthumbs.db
 Thumbs.db
 *.swp
+.vscode
```

Comment:
I will remove this. Artifact of adding all changes.

Reply:
This is I added and already created PR upstream. 
[725ff4e](https://github.com/digantdesai/XNNPACK/pull/6/commits/725ff4e1cc503a4ec788d0592d80fcb27a03da9b)

Reply:
Sounds good. I thought vocode somehow auto added it out something.